changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,23 @@
 {
-  "changelog": [
+  "changelog": [	
+    {
+      "ChangeList": [
+        {
+          "Id": "b94c9471-549a-4d52-83fc-f639c1f6bd11",
+          "ApiChange": "Property",
+          "ChangedApiName": "isStateSessionEnabled",
+          "ChangeType": "Addition",
+          "Description": "Added the **isStateSessionEnabled** property to [onPremisesPublishing](https://docs.microsoft.com/en-us/graph/api/resources/onPremisesPublishing?view=graph-rest-beta) resource.",
```

Suggestion to enhance the changelog.
```suggestion
          "Description": "Added the **isStateSessionEnabled** property to the [onPremisesPublishing](https://docs.microsoft.com/en-us/graph/api/resources/onPremisesPublishing?view=graph-rest-beta) resource. Use this property to enable or disable validation of the **state** parameter for apps that use the OAuth 2.0 authorization code grant flow.",
```